cmd/kubelet/app/server.go

```diff
@@ -152,21 +153,24 @@ HTTP server: The kubelet can also listen for HTTP and respond to a simple API
 		Run: func(cmd *cobra.Command, args []string) {
 			// initial flag parse, since we disable cobra's flag parsing
 			if err := cleanFlagSet.Parse(args); err != nil {
+				klog.ErrorS(err, "Failed to parse kubelet flag")
 				cmd.Usage()
-				klog.Fatal(err)
+				os.Exit(255)
 			}
 
 			// check if there are non-flag arguments in the command line
 			cmds := cleanFlagSet.Args()
 			if len(cmds) > 0 {
 				cmd.Usage()
-				klog.Fatalf("unknown command: %s", cmds[0])
+				klog.InfoS("Unknown command", "command", cmds[0])
```

Comment:
Fatal calls should use `ErrorS`, you can pass it a nil error rather than use `InfoS`: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#using-errors

/cc @serathius